pkg/pipelines/bootstrap.go

```diff
@@ -6,12 +6,35 @@ import (
 	"os"
 	"path"
 
+	v1rbac "k8s.io/api/rbac/v1"
+
 	"github.com/mitchellh/go-homedir"
 	"github.com/openshift/odo/pkg/pipelines/eventlisteners"
 	"github.com/openshift/odo/pkg/pipelines/routes"
 	"sigs.k8s.io/yaml"
 )
 
+var (
+	dockerSecretName = "regcred"
+	saName           = "demo-sa"
+	roleName         = "tekton-triggers-openshift-demo"
```

Comment:
Should we come up with better names for these now?

Reply:
We should.  We need to revisit many names and other hardcoded items.   We probably won't address them in this PR though.